coding-challenges/coding-challenge-1/tests.spec.js

```diff
@@ -1,8 +1,10 @@
-// import { pluralizeWords } from "./coding-challenge-1";
+import { pluralizeWords } from "./coding-challenge-1";
```

Comment:
I would also make a change to the filename, since there is already a filename with the name "tests.spec.js"